tests/wallets/.env.example

```diff
@@ -0,0 +1,2 @@
+# BlinkWallet
+export BLINK_TOKEN="your_token_here"
```

Comment:
this file is wrong, it should be added to `.env.example` in root directory

Reply:
or is this needed for the tests?

Reply:
file removed